cql3/query_processor.cc

```diff
@@ -103,7 +103,7 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, qu
     qp_group.push_back(sm::make_derive(
         "statements_prepared",
         _stats.prepare_invocations,
-        sm::description("Counts a total number of parsed CQL requests.")));
+        sm::description("Counts the total number of parsed CQL requests.")));
```

Comment:
Put the change that edits descriptions to a separate commit, and describe what's the nature of the change in the commit message (improving grammar or something).

Reply:
Sure, I'll do it, but...
While I understand the reason, I'm afraid that strict enforcing of such a rule might discourage future Boy Scouts. I'm not saying it's wrong, I'm just drawing the attention to something that might need more thought.

Reply:
True that, I myself have many times resigned from doing cosmetic changes like this due to the need to create a separate commit.
However when I feel a change is very important, I still do it (properly, with a commit).
But if I get a "meh, don't care so much" feeling, that tells me I shouldn't have wasted any thought on it in the first place (just my interpretation, I know a lot of people would disagree).

Reply:
It doesn't have to be a separate PR. Just a separate commit. Shouldn't be too hard and discourage anyone :).